packages/storage/src/service.ts

```diff
@@ -324,6 +326,7 @@ export class FirebaseStorageImpl implements FirebaseStorage {
     appCheckToken: string | null,
     retry = true
   ): Request<O> {
+    updateEmulatorBanner('Service', this._isUsingEmulator);
```

Comment:
did you mean to get rid of this so we don't update on every call?

Reply:
Ah, thanks for catching this!